src/redis_ipc.c

```diff
@@ -381,7 +379,7 @@ static json_object * redis_pop(const char *queue_path, unsigned int timeout)
 
     // parse popped entry back into json object
     entry = json_tokener_parse(json_text);
-    if (is_error(entry)) entry = NULL;
+    if (!entry) entry = NULL;
 
```

Comment:
If json funcs nowadays just return NULL instead of an error that needs interpreted, just remove lines like this